model/src/main/kotlin/Mappers.kt

```diff
@@ -51,3 +51,5 @@ private val mapperConfig: ObjectMapper.() -> Unit = {
 val jsonMapper = ObjectMapper().apply(mapperConfig)
 val xmlMapper = ObjectMapper(XmlFactory()).apply(mapperConfig)
 val yamlMapper = ObjectMapper(YAMLFactory()).apply(mapperConfig)
+
+val EMPTY_NODE = jsonMapper.readTree("{}")
```

Comment:
Given that the above are camel-case, should this one, too?

Reply:
How about an extension property instead, so it's more like e.g. in our own `Package` class?

    val ObjectMapper.EMPTY: JsonNode
        get() = jsonMapper.readTree("{}")

On the other hand, for `xmlMapper` this would then return a `JsonNode`, which is probably not what we want. Which brings me to the next question, should this field be named `EMPTY_JSON_NODE`?

Reply:
I didn't want to create a new node each time and extension properties cannot have backing fields. Also all mappers use the same API, so `xmlMapper` also uses `JsonNode`. Renaming done.